plugins/woocommerce-admin/client/products/sections/product-details-section.tsx

```diff
@@ -23,8 +47,20 @@ export const ProductDetailsSection: React.FC = () => {
 		>
 			<TextControl
 				label={ __( 'Name', 'woocommerce' ) }
-				name="name"
-				{ ...formContext.getInputProps< string >( 'name' ) }
+				name={ `${ PRODUCT_DETAILS_SLUG }-name` }
+				placeholder={ __( 'e.g. 12 oz Coffee Mug', 'woocommerce' ) }
+				{ ...getInputProps< string >( 'name' ) }
+			/>
+			<CheckboxControl
+				label={
+					<EnrichedLabel
+						label="Feature this product"
+						helpDescription="Include this product in a featured section on your website with a widget or shortcode."
```

Comment:
label and helpDescription should be wrapped with `__( '', 'woocommerce')`

Reply:
Nice catch! I fixed it in the commit 3829fb4